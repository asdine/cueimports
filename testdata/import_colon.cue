package file

altSize: alt.#AltDimensions & {
	w: 150
	h: 150
}
