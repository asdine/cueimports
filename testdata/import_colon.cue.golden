package file

import "test.com/dimensions:alt"

altSize: alt.#AltDimensions & {
	w: 150
	h: 150
}
